salvador/apiSecret.swift

```diff
@@ -0,0 +1,10 @@
+//
+//  apiSecret.swift
+//  salvador
+//
+//  Created by Geury Roustand on 07.02.23.
+//
+
+import Foundation
+
+var apiSecret = ""
```

Comment:
Instead of having to manually past the API key here I would recommend reading it from a JSON file that you don't check into the git repo.